crates/db_schema/src/utils.rs

```diff
@@ -59,8 +58,13 @@ pub const SITEMAP_LIMIT: i64 = 50000;
 pub const SITEMAP_DAYS: Option<TimeDelta> = TimeDelta::try_days(31);
 pub const RANK_DEFAULT: f64 = 0.0001;
 
+/// Some connection options to speed up queries
+const CONNECTION_OPTIONS: [&str; 1] = ["geqo_threshold=12"];
 pub type ActualDbPool = Pool<AsyncPgConnection>;
 
+/// Use a once_lock to create the postgres connection config, since this config never changes
+static POSTGRES_CONFIG_WITH_OPTIONS: OnceLock<String> = OnceLock::new();
```

Comment:
I'm not super familiar with how these work, but I assume they all need to be static / top level so they don't keep getting re-initialized.

Reply:
No you can put it directly inside the function (and the const above also).

https://stackoverflow.com/q/70230937

Reply:
Done